config/environments/production.rb

```diff
@@ -48,7 +48,7 @@
   # config.action_cable.allowed_request_origins = [ "http://example.com", /http:\/\/example.*/ ]
 
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
-  config.force_ssl = true
+  config.force_ssl = ENV.key?('DISABLE_HTTPS') ? false : true
```

Comment:
this line scares me

Reply:
Should be fine. In cloud deployed apps, anything coming through the ingress will redirect to https, regardless of this rails config flag. But is important to be able to bypass it for docker-compose.

https://kubernetes.github.io/ingress-nginx/user-guide/tls/#server-side-https-enforcement-through-redirect

> By default the controller redirects HTTP clients to the HTTPS port 443 using a 308 Permanent Redirect response if TLS is enabled for that Ingress.